openedx/features/course_experience/api/v1/views.py

```diff
@@ -57,6 +57,10 @@ def reset_course_deadlines(request):
 
     IMPORTANT NOTE: If updates are happening to the logic here, ALSO UPDATE the `reset_course_deadlines`
     function in common/djangoapps/util/views.py as well.
+    As of July 15, 2021, the logic has a slight difference in that this endpoint ignores
+    missed_deadlines due to learners being able to shift deadlines at the problem level regardless
+    of any missed_deadlines. The other endpoint isn't used for shifting due dates at the problem
+    level in the Learning MFE (which is now the default experience).
```

Comment:
Would it hurt to change other endpoint to match this new logic? Or do we like that extra check over there?